arithmetization/src/main/java/net/consensys/linea/extradata/LineaExtraDataPlugin.java

```diff
@@ -67,9 +67,34 @@ public void doRegister(final BesuContext context) {
   public void start() {
     super.start();
     if (profitabilityConfiguration.extraDataPricingEnabled()) {
+      final var extraDataHandler =
+          new LineaExtraDataHandler(rpcEndpointService, profitabilityConfiguration);
+      final var chainHeadHeader = blockchainService.getChainHeadHeader();
+      final var initialExtraData = chainHeadHeader.getExtraData();
```

Comment:
Is it possible to make a test to cover this? If I understand correctly, this ensures continuity in extraData handling, I.e. if Besu is restarted, it will take extraData from the last known block, rather than use the default values, right?

If it's going to be too much effort, I think it's fine leaving it as is

Reply:
Need to add support for this in the ATs, but I think it is worth the effort